### DISCUSSION TITLE
Is k8s necessary for GraphScope

### DISCUSSION THREAD

**Discussion by yecol** [alibaba/GraphScope#113] [Q&A] [answered]
**original issue title: a publicly available k8s cluster on aliyun as playground for tutorials and quick start**

As titled. Let's provide a cluster as a playground for beginners. 

Since the original issue will be solved via #65, and a question posed as below, the title changed to "Is k8s is a necessity for GraphScope"

**Comment by Zarca** [comment]
Hi,
I' m wondering if the K8S is a necessity that the GraphScope must be used along with a Kubernetes cluster?
As the documentation infered , dose it means that the GraphScope's coordinator will not work without the K8S?
So do the Vineyard?

**Comment by yecol** [comment]
Hi Zarca, 
Currently, GraphScope is distributed as docker image and the coordinator is designed to be launched in the k8s cluster.
k8s is unnecessary for vineyard. vineyard can be run in local or in a cluster of machines with an etcd for syncing metadata.

**Comment by Zarca** [comment]
Hi yecol,
Did you mean thay the major components of GraphScope ,like coordinator, GAE，GIE, GLE, and also Vineyard are all currently running as bunch of pods automatically in k8s behind?
In that case, is that the GraphScope's python wheel (installed with pip3) just a laucher or a client to the GraphScope's Docker Image?

**Comment by yecol** [comment]
Yes, exactly.

**Comment by Zarca** [comment]
Got it, thanks a lot :)

**Comment by yecol** [comment, marked as answer]
Will solve this via #65
